Makefile

```diff
@@ -79,7 +79,7 @@ version: ## Create new version of packages
 # @see https://github.com/lerna/lerna/tree/main/commands/version#readme
 # Bump version of packages changed since the last release
 # --yes` - skip all confirmation prompts
-	$(PKG_MANAGER) $(MONOREPO_TOOL) version --yes --no-push $(MONOREPO_TOOL_FLAGS) $(MONOREPO_TOOL_NO_PUSH)
+	$(PKG_MANAGER) $(MONOREPO_TOOL) version --yes --no-push --create-release github $(MONOREPO_TOOL_FLAGS) $(MONOREPO_TOOL_NO_PUSH)
```

Comment:
https://github.com/lerna/lerna/tree/main/libs/commands/version#--create-release-type 
> Requires --conventional-commits to be passed so that changelogs can be generated.

Do we do that?

Reply:
Yes, see https://github.com/lmc-eu/spirit-design-system/blob/main/lerna.json#L10

Reply:
And moved to the change to `lerna.json`.